apps/web/src/components/docs/Docs.tsx

```diff
@@ -22,29 +24,69 @@ const getMDXComponent = mdxBundler.getMDXComponent;
 
 const DOCS_WRAPPER_ELEMENT_ID = 'embedded-docs';
 
+const globalOverrides = {
+  frameworkterminal: {
+    FrameworkTerminal: () => <nv-framework-terminal></nv-framework-terminal>,
+  },
+  framework: () => null,
+  echoterminal: {
+    EchoTerminal: () => <nv-framework-terminal></nv-framework-terminal>,
+  },
+};
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please add a comment explaining what these do / why they are needed‏

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is 'echo' here our own keyword? If so, let's update it to match our new terminology‏

Reply:
Right now we need 'echo' as that file in our docs repo have not been renamed yet

Reply:
Can we please add a TODO in the code then that mentions the legacy name and why it's still there?